run-tests.js

```diff
@@ -275,9 +275,9 @@ async function main() {
 
       children.add(child)
 
-      child.on('exit', async (code) => {
+      child.on('exit', async (code, signal) => {
         children.delete(child)
-        if (code) {
+        if (code || signal !== null) {
```

Comment:
What a great catch! I would suggest the condition be `if (code !== 0)`, which to me communicates the intent very slightly better.